src/dataviews/dataview-model-base.js

```diff
@@ -98,9 +99,21 @@ module.exports = Model.extend({
     this._setupAnalysisStatusEvents();
   },
 
+  set: function (attributes, options) {
```

Comment:
I dont understand this method :S

Reply:
It's overriding Backbone's `set` to check that `source` is an AnalysisModel in case somebody takes the backdoor of using `set` after the widget is created. As talked with Pablo, this makes more sense in `update` function.